services/button.md

```diff
@@ -23,9 +23,10 @@ The amount of time a button needs to be held for a `hold` event to be generated.
 
 Emitted when button goes from inactive (`pressed == 0`) to active.
 
-    event up @ inactive
+    event up { time: u32 ms } @ inactive
```

Comment:
i would say that u16 is enough here. where do we want to cap the time? how much can padauk handle?

We also have the 8ms unit if needed.